src/util/asset/assetUtils.test.ts

```diff
@@ -0,0 +1,45 @@
+import {describe, expect, test} from "@jest/globals";
+
+import {ALGO_ASSET_ID} from "./assetConstants";
+import {isAlgo, prepareAssetPairData, sortAssetIds} from "./assetUtils";
+
+test("`sortAssetIds` returns an array with bigger id first", () => {
```

Comment:
Maybe we can add additional checks to the sortAssetIds function and their tests, what do you think? Here are the checks I suggest:
```tsx
  if (asset1ID === asset2ID) {
    throw new Error("asset1ID and asset2ID must be different");
  }

  // If I pass NaN to an asset id, Math.min or Math.max will always return the NaN, this check prevents it too.
  if (asset1ID < 0 || asset2ID < 0) {
    throw new Error("asset1ID and asset2ID must be positive integer");
  }
```

<img width="291" alt="image" src="https://user-images.githubusercontent.com/43711824/219666578-20bd8453-a9e6-4eea-a537-23cfa6697645.png">

Reply:
yeah It can be nice, I'll create a ticket for this